src/sst/elements/osseous/stream/mlmstream_bak.c

```diff
@@ -0,0 +1,169 @@
+// Copyright 2009-2015 Sandia Corporation. Under the terms
```

Comment:
Double checking you meant to include this file since it's called '_bak' ?

Reply:
Cleaned up in 84416d6